azurerm/data_source_key_vault.go

```diff
@@ -26,17 +26,9 @@ func dataSourceArmKeyVault() *schema.Resource {
 
 			"location": azure.SchemaLocationForDataSource(),
 
-			"sku": {
```

Comment:
Not in the data source, since it is read only... when it writes to the state file it will persist the new attribute instead of the deprecated attribute to be consistent.

Reply:
As about we should be setting both `sku` and `sku_name` here